src/views/home.hbs

```diff
@@ -14,5 +14,11 @@
      <input type="submit" value="Submit">
    </form>
 </section>
-
 </header>
+
+<section>
+  {{#each data}}
+  <img src="{{url}}">
+  <h1>des{{description}}</h1>
```

Comment:
I would make this black so you can see the descriptions. I would also change them from a h1 or h2 or paragraph as there should only be one h1 on the page.